src/libs/kata-types/tests/texture/configuration-anno-0.toml

```diff
@@ -84,6 +84,7 @@ sandbox_bind_mounts=["/proc/self"]
 vfio_mode="vfio"
 experimental=["a", "b"]
 enable_pprof = true
+name="virt-container"
```

Comment:
Why is this change needed? Can you add some explanation in the commit message?

Reply:
Ok, comments to be added. The explanation is , it will return error if no such name setting when do 
``` code
update_config_by_annotation {
    ...
if config.runtime.name.is_empty() {
            return Err(io::Error::new(
                io::ErrorKind::InvalidData,
                "Runtime name is missing in the configuration",
            ));
        }
...
}
```

But I am curious about our CIs no-run with UT ?

Reply:
Ok, thanks for the explanation and updating the commit message. Yes, I am a bit surprised why the CI did not catch this. Can you look into it?

Reply:
Yes, I will spend some time to check why CI doesn't run into these parts. And I will sync with you.